taca/testing/cli.py

```diff
@@ -0,0 +1,72 @@
+
+""" CLI for the testing commands
+"""
+import os
+import click
+import taca.testing.create_uppmax_like_env as createupp
+
+@click.group()
+def uppmax_env():
+    """ Create a local set of folders that resembles the uppmax-ngi env. Creates config file for ngi_pipeline, taca, and taca ngi-pipeline. Only a minimal taca config is needed (statusdb and log) 
+        The condig file (in general saved in variable NGI_CONFIG needs to looks something similar to:
+        
+        \b
+        environment:
+            project_id: ngi1234 #CAN BE ANYTHING
+            ngi_scripts_dir: /Users/vezzi/opt/ngi_pipeline/scripts #CAN BE ANYTHING
+            conda_env: TACA #CAN BE ANYTHING
+        flowcell_inbox:
```

Comment:
Seems like in the code you are trying to fetch "flowcell_inbox" under "environment" section. So probably missed an "indent" here